### DISCUSSION TITLE
Particle effect for snow fire/flame?

### DISCUSSION THREAD

**Discussion by chengenzhao** [AlmasB/FXGL#1110] [Q&A]
Hi:

Do we have samples for particle effects of snow, flame/fire?

Thanks

**Comment by AlmasB** [comment]
Hi,

there's rain (reference implementation): https://github.com/AlmasB/FXGL/tree/dev/fxgl-samples/src/main/java/intermediate/particles

there's smoke/fire (sandbox, so code is WIP / dodgy): https://github.com/AlmasB/FXGL/tree/dev/fxgl-samples/src/main/java/sandbox/particles

would be good to have a snow one...
